src/en-GB/news/2021/v15-2-10-octopus-released/index.md

```diff
@@ -0,0 +1,109 @@
+    
+- mgr/dashboard/monitoring: upgrade Grafana version due to CVE-2020-13379 ([pr#39306](https://github.com/ceph/ceph/pull/39306), Alfonso Martínez)
+    
+- mgr/dashboard: CLI commands: read passwords from file ([pr#39436](https://github.com/ceph/ceph/pull/39436), Ernesto Puerta, Alfonso Martínez, Juan Miguel Olmo Martínez)
+    
+- mgr/dashboard: Fix for incorrect validation in rgw user form ([pr#39027](https://github.com/ceph/ceph/pull/39027), Nizamudeen A)
+    
+- mgr/dashboard: Fix missing root path of each session for CephFS ([pr#39868](https://github.com/ceph/ceph/pull/39868), Yongseok Oh)
+    
+- mgr/dashboard: Monitoring alert badge includes suppressed alerts ([pr#39512](https://github.com/ceph/ceph/pull/39512), Aashish Sharma)
+    
+- mgr/dashboard: add ssl verify option for prometheus and alert manager ([pr#39872](https://github.com/ceph/ceph/pull/39872), Jean "henyxia" Wasilewski)
+    
+- mgr/dashboard: avoid using document.write() ([pr#39527](https://github.com/ceph/ceph/pull/39527), Avan Thakkar)
+    
+- mgr/dashboard: delete EOF when reading passwords from file ([pr#40155](https://github.com/ceph/ceph/pull/40155), Alfonso Martínez)
+    
+- mgr/dashboard: fix MTU Mismatch alert ([pr#39854](https://github.com/ceph/ceph/pull/39854), Aashish Sharma)
+    
+- mgr/dashboard: fix issues related with PyJWT versions >=2.0.0 ([pr#39836](https://github.com/ceph/ceph/pull/39836), Alfonso Martínez)
+    
+- mgr/dashboard: fix tooltip for Provisioned/Total Provisioned fields ([pr#39645](https://github.com/ceph/ceph/pull/39645), Avan Thakkar)
+    
+- mgr/dashboard: prometheus alerting: add some leeway for package drops and errors ([pr#39507](https://github.com/ceph/ceph/pull/39507), Patrick Seidensal)
+    
+- mgr/dashboard: report mgr fsid ([pr#39852](https://github.com/ceph/ceph/pull/39852), Ernesto Puerta)
+    
+- mgr/dashboard: set security headers ([pr#39627](https://github.com/ceph/ceph/pull/39627), Avan Thakkar)
+    
+- mgr/dashboard: trigger alert if some nodes have a MTU different than the median value ([pr#39103](https://github.com/ceph/ceph/pull/39103), Aashish Sharma)
+    
+- mgr/dashboard:minimize console log traces of Ceph backend API tests ([pr#39545](https://github.com/ceph/ceph/pull/39545), Aashish Sharma)
+    
+- mgr/rbd\_support: create mirror snapshots asynchronously ([pr#39376](https://github.com/ceph/ceph/pull/39376), Mykola Golub, Kefu Chai)
+    
+- mgr/rbd\_support: mirror snapshot schedule should skip non-primary images ([pr#39863](https://github.com/ceph/ceph/pull/39863), Mykola Golub)
+    
+- mgr/volume: subvolume auth\_id management and few bug fixes ([pr#39390](https://github.com/ceph/ceph/pull/39390), Rishabh Dave, Patrick Donnelly, Kotresh HR, Ramana Raja)
+    
+- mgr/zabbix: format ceph.\[{#POOL},percent\_used as float ([pr#39235](https://github.com/ceph/ceph/pull/39235), Kefu Chai)
```

Comment:
`{#POOL}` is causing an error, it can be resolved by removing the `#`.

@Thingee Do you know what this is used for, can it be removed?

Reply:
Done